src/helpers/model-helper.js

```diff
@@ -2,19 +2,41 @@ import helpers from './index';
 
 module.exports = {
   transformAttributes (flag) {
+    const dialect = helpers.version.getDialectName();
+    /*
+     * SpecialDataTypes is used to filter out values postgres, mysql, sqlite, mssql.
+     * It should be read from postgres instead of being enumerated.
+     */
+    const specialDataTypes = ['postgres', 'mysql', 'sqlite', 'mssql'];
+
+    /*
+     * Use either dataTypes for the dialect or dataTypes from Sequelize
+     */
+    const knownDataTypes = dialect 
```

Comment:
As dialect name is always defined other condition will never execute?

Reply:
Sure. The extra should be removed then. updating.